typescript/ecs/fargate-service-with-auto-scaling/package-lock.json

```diff
@@ -5,14 +5,12 @@
   "requires": true,
```

Comment:
Can you remove this file from the commit as it's generated when you run `npm install`

Reply:
d7f2e8a37ecd8caa1872e9e800f11d6b687374fe addresses this issue